src/main/java/frc/team2412/robot/commands/drivebase/DriveCommand.java

```diff
@@ -40,40 +40,38 @@ public class DriveCommand extends CommandBase {
 					.getEntry();
 	private static GenericEntry cubeSpeedEntry =
 			Shuffleboard.getTab("Drivebase")
-					.add("Cube Speed", true)
+					.addPersistent("Cube Speed", true)
 					.withWidget(BuiltInWidgets.kToggleSwitch)
 					.getEntry();
-	private static GenericEntry triggerModifierEntry =
+	private static GenericEntry turboRotationEntry =
 			Shuffleboard.getTab("Drivebase")
-					.add("Trigger Modifier", TRIGGER_MODIFIER_DEFAULT)
+					.addPersistent("Turbo Rotation Modifier", TURBO_ROTATION_DEFAULT)
 					.withWidget(BuiltInWidgets.kNumberSlider)
-					.withProperties(Map.of("Min", 0, "Max", 1))
+					.withProperties(Map.of("Min", 0.5, "Max", 2.5))
 					.getEntry();
 
 	public DriveCommand(
 			DrivebaseSubsystem drivebaseSubsystem,
 			DoubleSupplier forward,
 			DoubleSupplier strafe,
 			DoubleSupplier rotation,
-			DoubleSupplier speedLimiter) {
+			DoubleSupplier turboRotation) {
 		this.drivebaseSubsystem = drivebaseSubsystem;
 		this.forward = forward;
 		this.strafe = strafe;
 		this.rotation = rotation;
 		// this variable give the right trigger input
-		this.speedLimiter = speedLimiter;
+		this.turboRotation = turboRotation;
 
 		addRequirements(drivebaseSubsystem);
 	}
 
 	@Override
 	public void execute() {
-		// this is so ugly spotless
-		double driveSpeedModifier =
-				driveSpeedEntry.getDouble(1.0)
-						* (1
-								- (speedLimiter.getAsDouble()
-										* (1 - triggerModifierEntry.getDouble(TRIGGER_MODIFIER_DEFAULT))));
+		double rotationSpeedModifier =
+				rotationSpeedEntry.getDouble(1.0)
+						* (1 - (turboRotation.getAsDouble()
+										* (1 - turboRotationEntry.getDouble(TURBO_ROTATION_DEFAULT))));
```

Comment:
It seems strange to me that the value of `1 - turboRotationEntry.getDouble(TURBO_ROTATION_DEFAULT)` can be either positive or negative. Is this intentional?